org.eclipse.core.externaltools/src/org/eclipse/core/externaltools/internal/launchConfigurations/BackgroundResourceRefresher.java

```diff
@@ -73,18 +68,8 @@ public void handleDebugEvents(DebugEvent[] events) {
 	 * Submits a job to do the refresh
 	 */
 	protected void refresh() {
-		Job job= new Job(ExternalToolsProgramMessages.BackgroundResourceRefresher_0) {
-			@Override
-			public IStatus run(IProgressMonitor monitor) {
-				try {
-					RefreshUtil.refreshResources(fConfiguration, monitor);
-				} catch (CoreException e) {
-					ExternalToolsCore.log(e);
-					return e.getStatus();
-				}
-				return Status.OK_STATUS;
-			}
-		};
-		job.schedule();
+		Job.create(ExternalToolsProgramMessages.BackgroundResourceRefresher_0, monitor -> {
```

Comment:
I just verified that this change leads to the same behavior regarding failure reporting via message-box and error-log entry.